metadata/src/main/java/org/apache/kafka/controller/QuorumControllerMetrics.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.controller;
+
+import com.yammer.metrics.core.Gauge;
+import com.yammer.metrics.core.Histogram;
+import com.yammer.metrics.core.MetricName;
+import com.yammer.metrics.core.MetricsRegistry;
+
+
+public final class QuorumControllerMetrics implements ControllerMetrics {
+    private final static MetricName ACTIVE_CONTROLLER_COUNT = new MetricName(
+        "kafka.controller", "KafkaController", "ActiveControllerCount", null);
+    private final static MetricName EVENT_QUEUE_TIME_MS = new MetricName(
+        "kafka.controller", "ControllerEventManager", "EventQueueTimeMs", null);
```

Comment:
Good point. I will fix it so that EventQueueTimeMs has its original meaning.

For now, I have added a metric called EventQueueProcessingTimeMs which deals with processing time.  I do want to do the per-state tracking but I don't think we have time right now